scheds/rust/scx_rustland/src/bpf/main.bpf.c

```diff
@@ -572,14 +572,8 @@ void BPF_STRUCT_OPS(rustland_update_idle, s32 cpu, bool idle)
 	 * A CPU is now available, notify the user-space scheduler that tasks
 	 * can be dispatched.
 	 */
-	if (is_usersched_needed()) {
-		/*
-		 * Kick the CPU to make it immediately ready to accept
-		 * dispatched tasks.
-		 */
-		scx_bpf_kick_cpu(cpu, 0);
```

Comment:
Hmm... I wonder why this is. It *might* be useful to add an option to re-enable this, so that if ppl report stalls, they can try the option as a debug step.

Reply:
'Hm... even with v0.1.5 dropping `scx_bpf_kick_cpu()` doesn't seem to make any difference, any chance that maybe something changed in the kernel? (Going before v0.1.5 is a bit tricky because of the ABI change)

Reply:
I'm also wondering if the fact that now I'm always dispatching from `.dispatch()` with `SCX_DSQ_LOCAL_ON | cpu` (using the cpu selected by the user-space scheduler) implicitly kicks the target cpu, therefore the kick in `.update_idle()` is redundant.

Reply:
Kernel side hasn't changed. Yeah, `LOCAL_ON|cpu` will wake up the CPU.

Reply:
ok, I think everything makes sense now. I was using `SCX_DSQ_GLOBAL` for the scheduler and it wasn't waking up without the explicit kick cpu. Then I changed the scheduler to also use `LOCAL_ON|cpu`, so if that implicitly kicks the cpu we don't need the kick in update_idle anymore.

Reply:
Sounds great. No need to add the option then. That sounds like it should be reliable. Thanks.

Reply:
I'll update the commit message, so it's more clear why we dropped that.